apache/superset
# style: Database connect UI Polish - Step 1

### SUMMARY
Added styling to match the [figma specifications](https://www.figma.com/file/NbuTOMaUdv2GCBoDyihdLw/Database-Connection?node-id=1698%3A394658) for step 1 of the database connection form.

### SCREENSHOTS
<img width="501" alt="step1-dbconnect-fix" src="https://user-images.githubusercontent.com/55605634/121993564-4a7e8d80-cd69-11eb-9b7d-98dae0fbf93e.png">

### TESTING INSTRUCTIONS
From the Superset home page:
- Hover over "Data" in the upper navigation bar
- Click "Database"
- Click the "+ Database" button below the navigation bar and to the right
- Observe the styling of "Step 1" of the database connection form

### ADDITIONAL INFORMATION
